app/models/post.rb

```diff
@@ -0,0 +1,9 @@
+class Post < ApplicationRecord
+  belongs_to :user, optional: true
+  has_one :picture, as: :picturable
+
+  enum status: { public: 1, private: 2 }, _prefix: true
+
+  validates :body, length: { in: 2..1000 }
```

Comment:
why 2?
why not -2?
Those are called magic numbers/
I can't understand why those numbers were chosen.
At least create constants for min and max length size of the post